tests/RouterTest.php

```diff
@@ -72,12 +72,12 @@ public function testPublicWeb()
         $this->assertSame('http://dframeframework.com/deep/css/example.css', $this->router->publicWeb('css/example.css', 'deep/'));
     }
 
-    // public function testMakeUrl()
-    // {
-    //     $this->assertSame('http://dframeframework.com/page/index', $this->router->makeUrl('page/:page?page=index'));
-    //     $this->assertSame('http://test.com/page/index', $this->router->domain('test.com')->makeUrl('page/:page?page=index'));
-    //     $this->assertSame('http://test.dframeframework.com/page/index', $this->router->subdomain('test')->makeUrl('page/:page?page=index'));
-    //     $this->assertSame('https://dframeframework.com/page/index', $this->router->setHttps(true)->makeUrl('page/:page?page=index'));
-    //     $this->assertSame('http://dframeframework.com/page/index', $this->router->makeUrl('page/:page?page=index'));
-    // }
+    public function testMakeUrl()
+    {
+        $this->assertSame('http://dframeframework.com/page/index', $this->router->makeUrl('page/:page?page=index'));
+        $this->assertSame('http://test.com/page/index', $this->router->domain('test.com')->makeUrl('page/:page?page=index'));
+        $this->assertSame('http://test.dframeframework.com/page/index', $this->router->subdomain('test')->makeUrl('page/:page?page=index'));
+        $this->assertSame('https://dframeframework.com/page/index', $this->router->setHttps(true)->makeUrl('page/:page?page=index'));
+        $this->assertSame('http://dframeframework.com/page/index', $this->router->setHttps(false)->makeUrl('page/:page?page=index'));
```

Comment:
The `$this->router` class instance has been set the `https` is true.

We need to revert this value and set the `https` is false.

Reply:
@peter279k  Yes Https(true) should after usage back to previous state